lib/project_types/theme/messages/messages.rb

```diff
@@ -32,6 +32,26 @@ module Messages
             APP
           },
         },
+        push: {
+          error: {
+            pushing_error: "Theme files couldn't be pushed to Shopify",
+            removing_error: "Theme files couldn't be removed from Shopify",
+          },
+          help: <<~HELP,
+            {{command:%s push}}: Replaces theme files on Shopify with those in your directory. If filenames are provided, only those files will be replaced; otherwise, the whole theme is replaced.
```

Comment:
```suggestion
            {{command:%s push}}: Uploads your local theme files to Shopify, overwriting the remote versions. If you specify filenames, only those files will be replaced. Otherwise, the whole theme will be replaced.
```

How does the user `provide`/`specify` filenames? relative directory/file strings separated by spaces? Might want to consider spelling this out.